arch/arm/soc/nordic_nrf/include/nrf_common.h

```diff
@@ -62,29 +62,29 @@
 #define NRF52_IRQ_FPU_IRQn                     38
 
 /**
- * @file UART baudrate divisors for NRF51/NRF52 family processors.
+ * @file UART baudrate divisors for nRF5x family processors.
  *
  * Based on Nordic MDK included header file: nrf52_bitfields.h
  * Uses the UARTE_BAUDRATE macros since they are more precise.
  */
 
-#define NRF5_UART_BAUDRATE_300                 0x00014000
-#define NRF5_UART_BAUDRATE_600                 0x00027000
-#define NRF5_UART_BAUDRATE_1200                0x0004f000
-#define NRF5_UART_BAUDRATE_2400                0x0009d000
-#define NRF5_UART_BAUDRATE_4800                0x0013b000
-#define NRF5_UART_BAUDRATE_9600                0x00275000
-#define NRF5_UART_BAUDRATE_14400               0x003af000
-#define NRF5_UART_BAUDRATE_19200               0x004ea000
-#define NRF5_UART_BAUDRATE_28800               0x0075c000
-#define NRF5_UART_BAUDRATE_38400               0x009d0000
-#define NRF5_UART_BAUDRATE_57600               0x00eb0000
-#define NRF5_UART_BAUDRATE_76800               0x013a9000
-#define NRF5_UART_BAUDRATE_115200              0x01d60000
-#define NRF5_UART_BAUDRATE_230400              0x03b00000
-#define NRF5_UART_BAUDRATE_250000              0x04000000
-#define NRF5_UART_BAUDRATE_460800              0x07400000
-#define NRF5_UART_BAUDRATE_921600              0x0f000000
-#define NRF5_UART_BAUDRATE_1000000             0x10000000
+#define NRF_UART_BAUDRATE_300                  0x00014000
```

Comment:
Are these baud-rate values "generic", i.e. not specific to nRF5x SOCs? If so, we need to update the comment in line 65

Reply:
They are generic to Nordic ICs

Reply:
Right, I will fix that

Reply:
Fixed

Reply:
I wonder: wouldn't it be better to use the MDK headers, when defining the BAUDRATEs? E.g.:
#define NRF_UART_BAUDRATE_9600                ((UARTE_BAUDRATE_BAUDRATE_Baud9600   << UARTE_BAUDRATE_BAUDRATE_Pos) & UARTE_BAUDRATE_BAUDRATE_Msk)

Reply:
maybe, but that would be outside of the scope of this PR

Reply:
I can agree with that.

Reply:
While not related, seems like these defines should move to be drivers/serial/uart_nrf5.c

Reply:
Yes, I agree. In fact I think they will disappear completely when we switch the serial driver to use our `nrfx` HAL.